src/components/flat-table/flat-table-row/flat-table-row.component.tsx

```diff
@@ -102,47 +96,94 @@ export const FlatTableRow = React.forwardRef<
     }: FlatTableRowProps,
     ref
   ) => {
-    const internalId = useRef(id ?? guid());
+    const internalId = useRef(String(id ?? guid()));
```

Comment:
thanks, of course it will, I'd somehow overlooked that 🤦‍♂️ 

In that case, I'd probably personally choose to write this as `useRef(id ? String(id) : guid())` but I admit that's not great either. I'll stop nitpicking over this 😅

Reply:
I'll go with this as I do prefer avoiding potentially converting a string into a string 👍